Library/ViewModels/FindFriendsFriendFollowCellViewModel.swift

```diff
@@ -135,6 +143,39 @@ public final class FindFriendsFriendFollowCellViewModel: FindFriendsFriendFollow
       )
       .skipRepeats()
 
+    self.followButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Follow_friend_name",
+          defaultValue: "Follow %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    self.unfollowButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Stop_following_friend_name",
+          defaultValue: "Stop following %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    let followedText = localizedString(
+      key: "Followed",
+      defaultValue: "Followed",
+      count: nil,
+      substitutions: [:])
+
+    let notFollowedText = localizedString(
+      key: "Not_followed",
+      defaultValue: "Not followed",
+      count: nil,
+      substitutions: [:])
+
+    self.cellAccessibilityLabel = Signal.combineLatest(self.name, self.projectsBackedText, isFollowed)
```

Comment:
Thank you @theginbin! I wasn't sure if  `accessibilityValue` will work in the right order with "auto" VoiseOver (following it vs. preceding it) but it totally did :) - I tested it and it worked!